frontend/src/bundles/common/pages/not-found/styles.module.scss

```
@@ -24,15 +31,17 @@
     }
 
     & .code {
-        padding: 60px 0 0;
-        font-size: 300px;
+        // font-size: max(calc(100px + 20vh), 300px);
+        font-size: max(20vh, 300px);
```

Not a blocker, but consider:
```suggestion
        font-size: clamp(100px, 25vw, 300px);
```